qiita_db/metadata_template/prep_template.py

```diff
@@ -111,7 +108,7 @@ def create(cls, md_template, raw_data, study, data_type,
 
         # Get a connection handler
         conn_handler = SQLConnectionHandler()
-        queue_name = "CREATE_PREP_TEMPLATE_%d" % raw_data.id
+        queue_name = "CREATE_PREP_TEMPLATE_%d" % study.id
```

Comment:
If you do not feel that is important and @josenavas thinks is fine, let's leave it as is ...